app/code/Magento/InventoryApi/Test/_files/stocks_with_sales_channel.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Fixture `app/code/Magento/InventorySalesApi/Test/_files/stock_website_sales_channels.php` seems to be similar. Please investigate if it's possible to use it.